provider/database.py

```diff
@@ -17,15 +17,30 @@
 import uuid
 
 from cloudant import Cloudant
+from cloudant import CouchDB
 from cloudant.result import Result
 
 class Database:
     db_prefix = os.getenv('DB_PREFIX', '')
     dbname = db_prefix + 'ow_kafka_triggers'
-    username = os.environ['CLOUDANT_USER']
-    password = os.environ['CLOUDANT_PASS']
+    username = os.environ.get('CLOUDANT_USER')
+    client = None
 
-    client = Cloudant(username, password, account=username)
+    if (username):
```

Comment:
This is the tricky part, since bluewhisk is injecting CLOUDANT_USER and CLOUDANT_PASSWORD.
If we get rid of them, the install scripts in blue should be revised as well.